src/Aspire.Dashboard/Components/Controls/PlotlyChart.razor.cs

```diff
@@ -174,7 +174,7 @@ private sealed class Trace
 
     private string FormatTooltip(string name, double yValue, DateTime xValue)
     {
-        return $"<b>{InstrumentViewModel.Instrument?.Name}</b><br />{name}: {FormatHelpers.FormatNumberWithOptionalDecimalPlaces(yValue, CultureInfo.CurrentCulture)}<br />Time: {FormatHelpers.FormatTime(xValue)}";
+        return $"<b>{InstrumentViewModel.Instrument?.Name}</b><br />{name}: {FormatHelpers.FormatNumberWithOptionalDecimalPlaces(yValue, CultureInfo.CurrentCulture)}<br />Time: {FormatHelpers.FormatTime(xValue, provider: CultureInfo.CurrentCulture)}";
```

Comment:
CurrentCulture is the default in these methods. My understanding is this parameter only exists for unit testing. Would be cleaner not to pass it here and elsewhere in the PR.

Reply:
Code analysis complains if it is missing. If there is an optional culture parameter or another overload with it then the analyzer wants you to be explicit.